t/maxconns.t

```diff
@@ -11,22 +11,25 @@ use FindBin qw($Bin);
 use lib "$Bin/lib";
 use MemcachedTest;
 
-plan skip_all => "maxconns test does not work";
-exit 0;
-
-plan tests => 6;
-
-# start up a server with 10 maximum connections
-my $server = new_memcached('-c 100');
-my $sock = $server->sock;
-my @sockets;
-
-ok(defined($sock), 'Connection 0');
-push (@sockets, $sock);
-
-
-foreach my $conn (1..10) {
-  $sock = $server->new_sock;
-  ok(defined($sock), "Made connection $conn");
-  push(@sockets, $sock);
+my $server = new_memcached('-c 30','','true');
```

Comment:
random "true" arguments are poor usability. just as a nitpick:
new_memcached_nodebug('-c 30'); - which would be a sub that just calls `new_memcached()` with the extra flag. something like that makes it more obvious to the test writer.

Reply:
good idea. fixed